src/extension/app-schema/app-schema-web/src/main/java/org/geoserver/complex/OutputFormatsSupport.java

```diff
@@ -0,0 +1,42 @@
+/* (c) 2020 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.complex;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import org.geoserver.platform.GeoServerExtensions;
+import org.geoserver.platform.Operation;
+import org.geoserver.wfs.WFSGetFeatureOutputFormat;
+import org.geoserver.wfs.response.ComplexFeatureAwareFormat;
+
+/**
+ * Provides high level function to check if the provided format is supported by complex to simple
+ * features conversion.
+ */
+public class OutputFormatsSupport {
+
+    /** Checks if the provided format is supported by complex to simple features conversion. */
+    public static boolean isSupported(String format, Object value, Operation operation) {
```

Comment:
Single method used only in the dispatcher callback, why roll a separate class for it?